### PR TITLE
49 table bug fix show reference docs

### LINKED ISSUE #49: Markdown tables not generated properly
Lack of newline after title causes tables to not be generated properly.

### PR DESCRIPTION
**Pull request by rdahlman3**
Adds the reference docs to the table of contents and readme so that they don't error when you try to visit. Also slightly edited the db reference doc to make it more consistent and hopefully clear, but open to suggestions on page name on both toc and the actual page

Closes #49

### CODE REVIEW

**Comment by rdahlman3** [comment]
@helmoski this branch is ready to be looked at now!

File: content/efcore/getting-started/tutorial-database-reference.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Database Tables 
+# Database Reference
+Below are the tables and data that these tutorials work with.
```
Comment (helmoski) [inline content/efcore/getting-started/tutorial-database-reference.md]: Let's not say "these tutorials". Technically, all of the EF Core content is a single tutorial. Let's word it something like, "Here is the data contained within the example database used throughout this tutorial".

Comment (helmoski) [reply content/efcore/getting-started/tutorial-database-reference.md]: Could you also update the description in the models page to say something similar?

File: content/efcore/getting-started/README.md [hunk 2]
```diff
@@ -18,3 +18,5 @@ This tutorial will cover the following topics:
 - [Querying Data](querying.md)
 - [Loading Related Data](loading-related-data.md)
 - [Manipulating Entities](manipulating-entities.md)
+- [Appendix: Database Reference](tutorial-database-reference.md)
+- [Appendix: Data Model Reference](tutorial-model-reference.md)
```
Comment (helmoski) [inline content/efcore/getting-started/README.md]: Let's put the data model reference first.

**Review by helmoski — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/toc.md [hunk 3]
```diff
@@ -3,3 +3,5 @@
 - [Querying Data](querying.md)
 - [Loading Related Data](loading-related-data.md)
 - [Manipulating Entities](manipulating-entities.md)
+- [Appendix: Database Reference](tutorial-database-reference.md)
+- [Appendix: Data Model Reference](tutorial-model-reference.md)
```
Comment (helmoski) [inline content/efcore/getting-started/toc.md]: model before database

File: content/efcore/getting-started/tutorial-model-reference.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
 # Entity Classes 
- 
-Here is what the entity classes that make up our example data model look like:
- 
+This document contains class definitions of the example data models used throughout this tutorial.
```
Comment (helmoski) [inline content/efcore/getting-started/tutorial-model-reference.md]: It's one single data model. The entity classes make up the data model. 
Maybe, "This document contains the entity class definitions for the example data model used throughout this tutorial."

**Review by helmoski — CHANGES_REQUESTED** [review]

**Pull request merged**